static/app/components/smartSearchBar/index.tsx

```diff
@@ -1281,12 +1276,10 @@ class SmartSearchBar extends React.Component<Props, State> {
 
   onAutoComplete = (replaceText: string, item: SearchItem) => {
     if (item.type === ItemType.RECENT_SEARCH) {
-      trackAnalyticsEvent({
-        eventKey: 'search.searched',
-        eventName: 'Search: Performed search',
-        organization_id: this.props.organization.id,
+      trackAdvancedAnalyticsEvent('search.searched', {
+        organization: this.props.organization,
         query: replaceText,
-        source: this.props.savedSearchType === 0 ? 'issues' : 'events',
```

Comment:
Great example of how how using TS for analytics prevents these kinds of bugs where we set the wrong fields :)